src/test/suite/install.test.ts

```diff
@@ -50,7 +50,11 @@ suite("ensure install methods are functioning as expected", function() {
 		installKubectlSpy.resetHistory();	
 	});
 
-	test("ensure cli version checking works correctly", async function() {
+	var testVar = test("ensure cli version checking works correctly", async function() {
+		if(process.env.VSCODE_CAMELK_GITHUB_TOKEN === undefined) {
```

Comment:
could be simplified by 
		if(!process.env.VSCODE_CAMELK_GITHUB_TOKEN) {

Reply:
it doesn't seem to be the same. The test is no more skipped: https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/666721723#L637

Reply:
which would mean that the env variable is somehow set

Reply:
effectively set in this build: https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/666721723#L187 don't understand why it is set as it is not on the cameltooling repository

Reply:
ok, there is a pending test when the env variable is not set: https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/666756002#L639

Reply:
that seems to be the correct behavior then. Anyways I am a bit concerned about the other exceptions in the test output....

`rejected promise not handled within 1 second: TypeError: Cannot read property 'catch' of undefined`

Reply:
that looks like we have somewhere an undefined value and try to access member vars which then causes an uncatched error which makes the promise time out

Reply:
seems to be in integrationExplorer.test.ts

Reply:
Unless you are pointing to a new one, this was not introduced by this PR.
see https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/664766935#L546 for a maste rbuild with this error

Reply:
ok